src/main/tv/codely/scala_http_api/module/course/domain/Course.scala

```diff
@@ -0,0 +1,14 @@
+package tv.codely.scala_http_api.module.course.domain
+
+import tv.codely.scala_http_api.module.shared.user.domain.UserId
+
+object Course {
+  def apply(id: String, title: String, lessons: BigDecimal, creatorId: String): Course = Course(
+    CourseId(id),
+    CourseTitle(title),
+    CourseLessons(lessons),
+    UserId(creatorId)
+  )
+}
+
+case class Course(id: CourseId, title: CourseTitle, lessons: CourseLessons, creatorId: UserId)
```

Comment:
`lessons` could be understood as the actual lessons representation with all their corresponding attributes. Maybe we could use `totalLessons` or something like that so it denotes that it's just the total number of course lessons instead of the actual lessons 🙂